Dockerfile

```diff
@@ -15,13 +15,19 @@ RUN pip install --upgrade setuptools
 RUN pip install pysam
 
 WORKDIR /opt/repos/uta/
+
+# Install dependencies
 COPY pyproject.toml ./
+RUN pip install -e .[dev]
```

Comment:
Two pip-installs is the best solution I could come up with to prevent reinstalling dependencies whenever src changed (for a faster development cycle using docker).